sys/include/net/gnrc/ipv6/blacklist.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
```

Comment:
Then why not just add another line with your name and the current year? But it's up to you (: